diff -ru a/vendor/wlf/codectools.py b/vendor/wlf/codectools.py
--- a/vendor/wlf/codectools.py	2022-06-22 15:20:31.461921000 +0800
+++ b/vendor/wlf/codectools.py	2022-06-22 15:25:30.993922000 +0800
@@ -7,8 +7,8 @@
 import os
 import sys
 
-import six
-from six import text_type
+from wulifang.vendor import six
+from wulifang.vendor.six import text_type
 
 
 def u_print(msg, **kwargs):
diff -ru a/vendor/wlf/decorators.py b/vendor/wlf/decorators.py
--- a/vendor/wlf/decorators.py	2022-06-22 15:20:31.463922000 +0800
+++ b/vendor/wlf/decorators.py	2022-06-22 15:25:54.613422000 +0800
@@ -11,7 +11,7 @@
 from multiprocessing.dummy import Queue
 from threading import Thread, current_thread
 
-import six
+from wulifang.vendor import six
 
 from .env import HAS_QT, has_nuke
 
diff -ru a/vendor/wlf/ffmpeg.py b/vendor/wlf/ffmpeg.py
--- a/vendor/wlf/ffmpeg.py	2022-06-22 15:20:31.466422000 +0800
+++ b/vendor/wlf/ffmpeg.py	2022-06-22 15:25:54.575422000 +0800
@@ -10,7 +10,7 @@
 from subprocess import PIPE
 from tempfile import mktemp
 
-import six
+from wulifang.vendor import six
 
 from .codectools import get_encoded as e
 from .codectools import get_unicode as u
diff -ru a/vendor/wlf/fileutil.py b/vendor/wlf/fileutil.py
--- a/vendor/wlf/fileutil.py	2022-06-22 15:20:31.466922000 +0800
+++ b/vendor/wlf/fileutil.py	2022-06-22 15:26:03.876922000 +0800
@@ -12,7 +12,7 @@
 import sys
 from subprocess import call
 
-from six.moves import urllib
+from wulifang.vendor.six.moves import urllib
 
 from .codectools import get_encoded as e
 from .codectools import get_unicode as u
diff -ru a/vendor/wlf/mimetools.py b/vendor/wlf/mimetools.py
--- a/vendor/wlf/mimetools.py	2022-06-22 15:20:31.467921000 +0800
+++ b/vendor/wlf/mimetools.py	2022-06-22 15:26:03.877422000 +0800
@@ -6,7 +6,7 @@
 
 import mimetypes
 
-from six import text_type
+from wulifang.vendor.six import text_type
 
 
 def same_mimetype(suffix_a, suffix_b):
diff -ru a/vendor/wlf/mp_logging.py b/vendor/wlf/mp_logging.py
--- a/vendor/wlf/mp_logging.py	2022-06-22 15:20:31.469422000 +0800
+++ b/vendor/wlf/mp_logging.py	2022-06-22 15:24:31.222922000 +0800
@@ -9,7 +9,7 @@
 import warnings
 from multiprocessing.dummy import Lock, Process, Queue
 
-from six import binary_type, text_type
+from wulifang.vendor.six import binary_type, text_type
 
 from .decorators import renamed
 from .path import get_unicode as u
diff -ru a/vendor/wlf/path.py b/vendor/wlf/path.py
--- a/vendor/wlf/path.py	2022-06-22 15:20:31.470422000 +0800
+++ b/vendor/wlf/path.py	2022-06-22 15:24:23.754422000 +0800
@@ -12,13 +12,13 @@
 import sys
 from functools import wraps
 
-import six
+from wulifang.vendor import six
 
 from .codectools import get_encoded, get_unicode
 from .pathtools import module_path
 
 if six.PY2:
-    import pathlib2 as pathlib  # pylint: disable=import-error
+    import wulifang.vendor.pathlib2_unicode as pathlib  # pylint: disable=import-error
 else:
     import pathlib  # pylint: disable=import-error
 
diff -ru a/vendor/wlf/progress/exceptions.py b/vendor/wlf/progress/exceptions.py
--- a/vendor/wlf/progress/exceptions.py	2022-06-22 15:20:31.480921000 +0800
+++ b/vendor/wlf/progress/exceptions.py	2022-06-22 15:25:54.577422000 +0800
@@ -4,7 +4,7 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-import six
+from wulifang.vendor import six
 
 
 @six.python_2_unicode_compatible
diff -ru a/vendor/wlf/progress/handlers/base.py b/vendor/wlf/progress/handlers/base.py
--- a/vendor/wlf/progress/handlers/base.py	2022-06-22 15:20:31.483422000 +0800
+++ b/vendor/wlf/progress/handlers/base.py	2022-06-22 15:25:54.613422000 +0800
@@ -7,7 +7,7 @@
 import time
 from datetime import timedelta
 
-import six
+from wulifang.vendor import six
 
 from ...codectools import u_print
 from ..exceptions import CancelledError
diff -ru a/vendor/wlf/progress/handlers/cli.py b/vendor/wlf/progress/handlers/cli.py
--- a/vendor/wlf/progress/handlers/cli.py	2022-06-22 15:20:31.484421000 +0800
+++ b/vendor/wlf/progress/handlers/cli.py	2022-06-22 15:25:54.613422000 +0800
@@ -6,7 +6,7 @@
 
 import sys
 
-import six
+from wulifang.vendor import six
 
 from .. import core
 from ...codectools import get_unicode as u
diff -ru a/vendor/wlf/progress/handlers/qt.py b/vendor/wlf/progress/handlers/qt.py
--- a/vendor/wlf/progress/handlers/qt.py	2023-06-29 11:17:16.544572000 +0800
+++ b/vendor/wlf/progress/handlers/qt.py	2023-06-29 11:21:53.443166100 +0800
@@ -4,8 +4,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from Qt import QtCompat, QtWidgets
-from Qt.QtCore import Qt, Signal
+from wulifang.vendor.Qt import QtCompat, QtWidgets
+from wulifang.vendor.Qt.QtCore import Qt, Signal
 
 from .. import core
 from ...codectools import get_unicode as u
diff -ru a/vendor/wlf/uitools/core.py b/vendor/wlf/uitools/core.py
--- a/vendor/wlf/uitools/core.py	2023-06-29 11:17:16.546522000 +0800
+++ b/vendor/wlf/uitools/core.py	2023-06-29 11:21:15.024852600 +0800
@@ -5,7 +5,7 @@
 
 import sys
 
-from Qt.QtWidgets import QAction, QApplication, QMenu
+from wulifang.vendor.Qt.QtWidgets import QAction, QApplication, QMenu
 
 from .. import mp_logging
 from ..progress.handlers.qt import QtProgressBar
diff -ru a/vendor/wlf/uitools/pyblish_l10n.py b/vendor/wlf/uitools/pyblish_l10n.py
--- a/vendor/wlf/uitools/pyblish_l10n.py	2023-06-29 11:17:16.546522000 +0800
+++ b/vendor/wlf/uitools/pyblish_l10n.py	2023-06-29 11:20:58.188334800 +0800
@@ -7,8 +7,8 @@
 import json
 from functools import wraps
 
-import pyblish.api  # pylint: disable=import-error
-from pyblish.plugin import discover  # pylint: disable=import-error
+import wulifang.vendor.pyblish.api  # pylint: disable=import-error
+from wulifang.vendor.pyblish.plugin import discover  # pylint: disable=import-error
 
 from ..pathtools import module_path
 from ..path import Path
diff -ru a/vendor/wlf/uitools/template/dialog_with_dir.py b/vendor/wlf/uitools/template/dialog_with_dir.py
--- a/vendor/wlf/uitools/template/dialog_with_dir.py	2023-06-29 11:17:16.549449000 +0800
+++ b/vendor/wlf/uitools/template/dialog_with_dir.py	2023-06-29 11:21:27.532300700 +0800
@@ -5,8 +5,8 @@
 
 from abc import abstractmethod
 
-from Qt import QtCompat, QtCore, QtWidgets
-from Qt.QtWidgets import QDialog, QFileDialog
+from wulifang.vendor.Qt import QtCompat, QtCore, QtWidgets
+from wulifang.vendor.Qt.QtWidgets import QDialog, QFileDialog
 
 from ...path import Path
 
diff -ru a/vendor/wlf/uitools/tray.py b/vendor/wlf/uitools/tray.py
--- a/vendor/wlf/uitools/tray.py	2023-06-29 11:17:16.547497000 +0800
+++ b/vendor/wlf/uitools/tray.py	2023-06-29 11:20:48.387336600 +0800
@@ -4,8 +4,8 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-from Qt.QtGui import QCursor, QIcon
-from Qt.QtWidgets import QSystemTrayIcon
+from wulifang.vendor.Qt.QtGui import QCursor, QIcon
+from wulifang.vendor.Qt.QtWidgets import QSystemTrayIcon
 
 from ..pathtools import module_path
 from .core import Menu
